bigtable/google/cloud/bigtable/instance.py

```diff
@@ -240,6 +241,44 @@ def create(self, location_id=_EXISTING_INSTANCE_LOCATION_ID,
             parent=parent, instance_id=self.instance_id, instance=instance,
             clusters=clusters)
 
+    def create_with_multiple_clusters(self, clusters):
```

Comment:
@tseaver, eventually, this should be the main implementation of `create()`.  How do you feel about turning `clusters` into a member variable, and using it in create().  It will require some validation that the optional parameters aren't used in conjunction with clusters, but ultimately, IMHO, will produce a better experience.